array_methods_lab.js

```diff
@@ -0,0 +1,371 @@
+// For spacing.
```

Comment:
Overall, great job! I see that you understand array methods. Why don't you try to work on the bonus problems? I'm sure you can handle them.